### PR TITLE
Backport: use preset-env and remove flow-strip-types

### PR DESCRIPTION
**Pull request by yavorsky**
Nothing special, just use `preset-env` for backport and remove `flow-strip-types` plugin.

### CODE REVIEW

**Comment by hzoo** [comment]
Is this necessary to backport though?

**Comment by yavorsky** [comment]
@hzoo For now it's still the main branch and few people asked me why we are using es2015 instead of env 😕

File: package.json [hunk 1]
```diff
@@ -76,12 +76,7 @@
   },
   "babel": {
     "presets": [
-      [
-        "es2015",
-        {
-          "loose": true
-        }
-      ]
+      ["env"]
```
Comment (tleunen) [inline package.json]: Isn't it better to use the current code instead of always the previous version?
So replacing `["env"]` with `"./src"`?

Comment (hzoo) [inline package.json]: If you have a bug then it won't build itself

**Comment by hzoo** [comment]
@yavorsky lol really..

**Pull request merged**